go.mod

```diff
@@ -84,8 +85,9 @@ require (
 	github.com/prometheus/client_golang v1.17.0
 	github.com/prometheus/client_model v0.5.0
 	github.com/prometheus/common v0.45.0
-	github.com/quay/claircore v1.5.19
+	github.com/quay/claircore v1.5.21-0.20231219014855-9210797e665e
```

Comment:
switched to `main` branch for testing purposes

Reply:
> switched to main branch for testing purposes

What did you need from main? If there is a chance it is going to be required for our release, maybe leave a comment.

Reply:
I definitely expect us to use an official ClairCore release for our release. v1.5.19 is several months old at this point, and we can't use v1.5.20 because it doesn't build on macOS. The main branch fixes that build issue and also has many updates. One of those is the newer way of fetching layers which had that memory issue (so it was good I tested with the main branch 😄)